src/str.js

```diff
@@ -742,6 +941,35 @@ Sk.builtin.str.prototype["isupper"] = new Sk.builtin.func(function (self) {
     return Sk.builtin.bool(self.v.length && !/[a-z]/.test(self.v) && /[A-Z]/.test(self.v));
 });
 
+Sk.builtin.str.prototype["istitle"] = new Sk.builtin.func(function (self) {
```

Comment:
Might be simpler to just compare the incoming value against the result of calling ```title()``` on that value?